hack/ci/Makefile

```diff
@@ -57,3 +59,22 @@ run-with-lifecycle-manager:
 .PHONY: run-without-lifecycle-manager
 run-without-lifecycle-manager:
 	@make -C ${PROJECT_COMMON} run-without-lifecycle-manager
+
+.PHONY: install-latest
+install-latest:
+	@make -C ${PROJECT_COMMON} kyma create-k3d
+	curl -LJ -s https://github.com/kyma-project/keda-manager/releases/latest/download/keda-manager.yaml > template-k3d.yaml
+	@make -C ${PROJECT_COMMON} install-module verify-kyma
+
+
+.PHONY: reinstall-keda
+reinstall-keda:
```

Comment:
Why did you move this target from common? IMO, the dependencies are very similar to run-with-lifecycle-manager and run-without-lifecycle-manager from common and this target beter fit there.

I see that you (probably) need depandency reinstall-keda in other target. You could use shortcut in this place:
```
reinstall-keda:
    @make -C ${PROJECT_COMMON} reinstall-keda
```

Reply:
Because it is only used by the CI targets

Reply:
ok